libnavigation-core/api/current.txt

```diff
@@ -35,20 +36,26 @@ package com.mapbox.navigation.core {
     method public void registerOffRouteObserver(com.mapbox.navigation.core.trip.session.OffRouteObserver offRouteObserver);
     method public void registerRoadObjectsOnRouteObserver(com.mapbox.navigation.core.trip.session.RoadObjectsOnRouteObserver roadObjectsOnRouteObserver);
     method public void registerRouteAlternativesObserver(com.mapbox.navigation.core.routealternatives.RouteAlternativesObserver routeAlternativesObserver);
+    method public void registerRouteAlternativesObserver(com.mapbox.navigation.core.routealternatives.NavigationRouteAlternativesObserver routeAlternativesObserver);
     method public void registerRouteProgressObserver(com.mapbox.navigation.core.trip.session.RouteProgressObserver routeProgressObserver);
     method public void registerRoutesObserver(com.mapbox.navigation.core.directions.session.RoutesObserver routesObserver);
     method public void registerTripSessionStateObserver(com.mapbox.navigation.core.trip.session.TripSessionStateObserver tripSessionStateObserver);
     method public void registerVoiceInstructionsObserver(com.mapbox.navigation.core.trip.session.VoiceInstructionsObserver voiceInstructionsObserver);
-    method public void requestAlternativeRoutes(com.mapbox.navigation.core.routealternatives.RouteAlternativesRequestCallback? callback = null);
```

Comment:
There are still overloads available that accept `RouteAlternativesRequestCallback` and `null`, so this doesn't break API compatibility.

Reply:
I think it is because we're missing a `@JvmOverloads` here https://github.com/mapbox/mapbox-navigation-android/blob/49791265ae60ff01679902dc0c0af3acd00db7bd/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L795

Reply:
It isn't needed because of the other overloads. Actually, it cannot even be set because it would collide with already available no-arg overload.